### ISSUE TITLE
import only works in source-tree

### ISSUE THREAD

**Issue by gdmcbain** [kinnala/scikit-fem#1]
Having forked & installed (in a Python 3.6 venv)
```shell
pip install -e git+https://github.com/gdmcbain/scikit-fem.git#egg=scikit-fem
```
I find that I can `import skfem` and run the examples in the source-tree but not out of it.  That is, inside:
```shell
cd ~/.pyenv/scikit-fem/src/scikit-fem
python examples/ex1.py
```
works fine but outside
```shell
cp examples/ex1.py ~
cd
python examples/ex1.py
```
> FileNotFoundError: [Errno 2] No such file or directory: 'skfem'

I think it's the `__name__` in `skfem/__init__.py` evaluating as `skfem`.

Maybe try something like
```python
for filename in os.listdir(os.path.dirname(__file__)):
```

**Comment by kinnala** [comment]
Thank you for the notification, the detailed description of the issue and the suggestions for fixing it. I will take a look at it before next week.

The slightly hacky content of __init__.py is a product of shear laziness. I used to list e.g. all different elements in __init__.py for automatic import: I often use the library for creating quick proof-of-concept solvers so automatic import saves time and I got annoyed while updating __init__.py each time after adding e.g. new elements.
